mail/notmuch/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           conflicts_build 1.0
 
 name                notmuch
 version             0.29
-revision            1
+revision            2
```

Comment:
It is actually not necessary to increase the revision for this, as it is just fixing builds that anyway previously failed. Those that succeeded do not need to run. So please set this back to 1.

Reply:
Point taken. Fixed.